Containerfile

```diff
@@ -495,3 +495,43 @@ RUN /tmp/image-info.sh && \
     mkdir -p /var/lib/bluetooth && \
     chmod -R 755 /var/lib/bluetooth && \
     ostree container commit
+
+FROM bazzite as bazzite-dx
+
+ARG IMAGE_NAME="${IMAGE_NAME}"
+ARG IMAGE_VENDOR="${IMAGE_VENDOR}"
+ARG IMAGE_FLAVOR="${IMAGE_FLAVOR}"
+ARG BASE_IMAGE_NAME="${BASE_IMAGE_NAME}"
+ARG FEDORA_MAJOR_VERSION="${FEDORA_MAJOR_VERSION}"
+
+COPY system_files/deck/shared system_files/deck/${BASE_IMAGE_NAME} /
+
+# Apply IP Forwarding before installing Docker to prevent messing with LXC networking
+RUN sysctl -p && \
+    wget https://github.com/docker/compose/releases/latest/download/docker-compose-linux-x86_64 -O /tmp/docker-compose && \
+    install -c -m 0755 /tmp/docker-compose /usr/bin
+
+COPY --from=cgr.dev/chainguard/flux:latest /usr/bin/flux /usr/bin/flux
+COPY --from=cgr.dev/chainguard/helm:latest /usr/bin/helm /usr/bin/helm
+COPY --from=cgr.dev/chainguard/ko:latest /usr/bin/ko /usr/bin/ko
+COPY --from=cgr.dev/chainguard/minio-client:latest /usr/bin/mc /usr/bin/mc
+
+# Install DevPod
+RUN rpm-ostree install https://github.com/loft-sh/devpod/releases/download/v0.3.7/DevPod_linux_x86_64.rpm && \
```

Comment:
I believe I read that this is going to be provided as an AppImage only going into the future with only the CLI being provided as an RPM. We could install the CLI, and then have a dx entry in the portal that provides downloading the AppImage as an option if that sounds all right to you?

Reply:
The plan was to COPR this one with the intent of seeing if someone can help us get it in fedora.

Reply:
I don't mind looking into throwing it on COPR. I think that's the best bet as it would be nice for it to just be there out of the box